CONTRIBUTION.md

```diff
@@ -0,0 +1,80 @@
+# Contribution (Draft)
+
+`Awesome Satori` is a resource list that showcases various interesting topics in the Satori community, aiming to collect and share excellent resources related to Satori, including protocol implementations, SDKs, frameworks, tutorials, tools, and more, to help others learn, discover, and use Satori.
+To ensure quality and consistency, this document will describe in detail how you should contribute your work.
+
+English | [简体中文](./CONTRIBUTION-zh.md)
+
+## Preparation
+
+Before contributing your work, we need some preparation:
+
+- Have a GitHub account and fork this repository.
+- Have some understanding or interest in Satori and be willing to provide valuable information and help to the community.
+- Ensure that the content you add is related to Satori and is of high quality, useful, interesting, or innovative.
+- Respect the original author's copyright and intellectual property rights and avoid adding any infringing or illegal content.
+
+## Adding Content
+
+This project welcomes you to add any excellent resources related to Satori, including but not limited to the following types:
+
+- Satori protocol implementations.
+- SDKs: Development tools based on the Satori protocol.
+- Frameworks: Frameworks or applications that use the Satori protocol for communication.
+- Tutorials: Guided articles or videos on how to use Satori and Satori protocol implementations SDKs/Frameworks for development, deployment, testing, debugging, etc.
+- Tools: Software, libraries, frameworks, plugins, etc. that help you improve your Satori development efficiency or experience.
+- Articles: In-depth or extensive articles on the principles, features, advantages, application scenarios, etc. of Satori.
+
+### Implementations, SDKs and Frameworks
+
+When adding `implementation/SDK/Framework` contribution content, we will make the following requirements:
+
+- Implementations, SDKs and Frameworks will be placed in tables separately with the following fields:
+   - `Name`: The name of your project, sorted by English alphabet A-Z (if not distinguished separately, official projects will be at the top)
+   - `Author`: The author name, which is you (should use name instead of username. Because name is unique on GitHub)
+   - `Language`: The language used to write the project. **This only indicates the language used to write the project**.
```

Comment:
```suggestion
- Implementations, SDKs and Frameworks will be placed in separate tables, with the following fields:
   - `Name`: Your project's name, sorted alphabetically from A-Z (official projects will be listed at the top if not individually distinguished).
   - `Author`: Your name, i.e., your GitHub account name (use name instead of username, as the name is unique on GitHub).
   - `Language`: The programming language in which the project is written; **This only indicates the language used to write the project.**
```